pkg/api/handlers/compat/networks.go

```diff
@@ -69,12 +70,14 @@ func convertLibpodNetworktoDockerNetwork(runtime *libpod.Runtime, network nettyp
 			return nil, err
 		}
 		if netData, ok := data.NetworkSettings.Networks[network.Name]; ok {
+			ipv4Address := fmt.Sprintf("%s/%d", netData.IPAddress, netData.IPPrefixLen)
+			ipv6Address := fmt.Sprintf("%s/%d", netData.GlobalIPv6Address, netData.GlobalIPv6PrefixLen)
```

Comment:
You have to check that `netData.IPAddress` is not an empty string before comping them to the CIDR. Right now you will return `/0` is there is no IP. It should just be empty.